enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -9,15 +9,40 @@ import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image
  * even when rendered under a React portal.
  */
 export const PublicComponentStylesWrapper = styled.div`
+  // Try to reset as much as possible to avoid css leaking from host app to our components
+  all: initial;
```

Comment:
These two line are a try to un-set things from the host app that we don't want.

Reply:
woah, TIL that this property exists. I'll try to test with Shoppy to see if there are some unintended consequences of this